### PR TITLE
(#64) Update analytics for Disease

### LINKED ISSUE #64: Update analytics for Disease
### Issue description
*Description of the issue*
We need to ensure other event is fired on the result item click. 
add this test:
```gherkin
Scenario: Click event fires when a user clicks on result item
        Given "trialListingPageType" is set to "Disease"
       And "pageTitle" is set to "{{disease_label}} Clinical Trials"
        And "analyticsPublishedDate" is set to "02/02/2011"
        When the user navigates to "/breast-cancer?cfg=0"
        Then the page title is "Breast Cancer Clinical Trials"
        When user clicks on result item 2
        Then there should be an analytics event with the following details
            | key              | value                             |
            | type             | Other                             |
            | event            | TrialListingApp:Other:ResultClick |
            | linkName         | CTSLink                           |
            | data.resultIndex | (int)2                            |
            | data.currentPage | (int)1                            |
```
update existing page load ( this should had already been updated)
```gherkin
 Scenario: Page Load Analytics fires when a user views a disease listing page
        Given "trialListingPageType" is set to "Disease"
        And "pageTitle" is set to "{{disease_label}} Clinical Trials"
        And "analyticsPublishedDate" is set to "02/02/2011"
        When the user navigates to "/C4872"
        Then the page title is "Breast Cancer Clinical Trials"
        And browser waits
        Then there should be an analytics event with the following details
            | key                                         | value                                                      |
	   | event                                       | TrialListingApp:Load:Results                               |
            | type                                        | PageLoad                                                   |
	| page.publishedDate                          | 02/02/2011                                                 |
            | page.contentGroup                           | Clinical Trials: Custom                                    |
	   | page.name                                   | www.cancer.gov/breast-cancer                               |
	  | page.title                                  | Breast Cancer Clinical Trials     							           |
	| page.channel                                | Clinical Trials                                            |
	 | page.type                                   | nciAppModulePage                                           |
	 | page.language                               | english                                                    |
	| page.metaTitle                              | Breast Cancer Clinical Trials - National Cancer Institute  |
	| page.additionalDetails.diseaseName			    | breast cancer		                                           |
            | page.additionalDetails.trialListingPageType | disease					                                           |
	| page.additionalDetails.numberResults        | (int)938                                                   |

```
> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  
2. 
3. 

### What's the expected result?
-

### What's the actual result?
-

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by alerosad**
* added click event test to DiseaseAnalytics.feature
Closes #64

### CODE REVIEW

**Comment by alerosad** [comment]
not sure why the pa11y tests aren't passing, they pass fine on my local machine

**Review by KateMashkinaNIH — APPROVED** [review]

**Comment by alerosad** [comment]
pa11y tests passing now, not sure why yesterday they were acting up

**Review by sarinapadilla — APPROVED** [review]

**Review by seyilonge-nci — APPROVED** [review]

File: cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature [hunk 1]
```diff
@@ -22,3 +22,18 @@ Feature: Disease listing page analytics
 			| page.additionalDetails.diseaseName          | breast cancer                                             |
 			| page.additionalDetails.trialListingPageType | disease                                                   |
 			| page.additionalDetails.numberResults        | (int)938                                                  |
+
```
Comment (arcepaul) [inline cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:25]: As discussed add the line when rebasing.

Comment (alerosad) [reply cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:26]: done

Comment (sarinapadilla) [reply cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:26]: Actually, this line needs to be added to the below test, with the click event - it's already been added to the first test with the previous commit to develop.

**Comment by alerosad** [comment]
it won't pass the DiseaseAnalytics.feature integration test, since I added: 
And "dynamicListingPatterns" object is set to "Disease"
to it. I added this since in Sarina's and Kate's PR they add this new parameter to their tests.

File: cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature [hunk 2]
```diff
@@ -22,3 +22,19 @@ Feature: Disease listing page analytics
...
+	Scenario: Click event fires when a user clicks on result item
+        Given "trialListingPageType" is set to "Disease"
+				And "dynamicListingPatterns" object is set to "Disease"
+       	And "pageTitle" is set to "{{disease_label}} Clinical Trials"
```
Comment (sarinapadilla) [inline cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature:29]: Please remove this line, as setting the dynamicListingPatterns on the line above takes care of it. pageTitle being set is only necessary for Manual pages now.

**Pull request merged**